src/ec/packages/ec-component-menu-legacy/menu-legacy.html.twig

```diff
@@ -0,0 +1,124 @@
+{% spaceless %}
+
+{#
+  Parameters:
+    - "label" (string) (default: ''): visible on mobile
+
+    - "items" (array) (default: []): [{
+        "label" (string): label of the link
+        "href" (string): target of the link
+        "isCurrent" (boolean): does the menu contains the current page? (default: false)
+        "children" (array) (default: []): [{
+          "items" (array) (default: [{}]): [{
+            - "label" (string): label of the link
+            - "href" (string): target of the link
+          }]
+        }]
+    }]
+      
+    - "extra_classes" (string) (default: '')
+    - "extra_attributes" (array) (default: []): format: [
+        {
+          "name" (string) (default: ''),
+          "value" (string) (default: '')
+        },
+        ...
+      ]
+
+#}
+
+{# Internal properties #}
+
+{% set _menu_label = label|default('') %}
+{% set _links = items|default([]) %}
+{% set _css_class = 'ecl-menu-legacy' %}
+{% set _prefix_class = 'ecl-menu-legacy' %}
+{% set _extra_attributes = '' %}
+
+{% set _icon = {
+  path: 'static/media/icons.a11fa655.svg',
+  type: '',
+  name: 'ui--corner-arrow',
+  size: 'xs',
+  transform: 'rotate-180',
+  color: '',
+  extra_classes:'ecl-menu-legacy_link-icon ecl-link__icon ecl-icon'}|merge(icon|default({})) 
+%}
+
+
+
+{# Internal logic - Process properties #}
+
+{% if extra_classes is defined and extra_classes is not empty %}
+  {% set _css_class = _css_class ~ ' ' ~ extra_classes %}
+{% endif %}
+
+{% if extra_attributes is defined and extra_attributes is not empty and extra_attributes is iterable %}
+  {% for attr in extra_attributes %}
+    {% set _extra_attributes = _extra_attributes ~ ' ' ~ attr.name ~ '="' ~ attr.value ~ '"' %}
+  {% endfor %}
+{% endif %}
+
+{# Print the result #}
+<nav data-ecl-auto-init="MenuLegacy" class="{{ _css_class }}"  data-ecl-menu-legacy="true" aria-expanded="false"  {{ _extra_attributes|raw }}>
+  <div class="ecl-container">
+    <button class="{{ _prefix_class }}__toggle {{ _prefix_class }}__hamburger  ecl-button ecl-button--ghost" type="button"  data-ecl-menu-legacy-hamburger-button="true" >
+      <div class="ecl-menu-legacy__toggle-container">
```

Comment:
these classes name should use `_prefix_class` to be consistent (that's the same for some other classes in this file)
or you don't use a prefix at all, it's up to you :)